src/components/ui/PopOver/index.tsx

```diff
@@ -37,15 +38,13 @@ export interface PopOverProps
   /** Defines the placement of PopOver menu. */
   placement?: Placement;
   /** Defines the function to render the inner contents of the popover button element */
-  renderButton?: (isActive: boolean) => {};
+  renderButton?: (isActive: boolean) => ReactNode;
```

Comment:
Does ReactNode also include `{}`, wondering if this will be a breaking change?

Reply:
If I don't change this type I get this error:
```
No overload matches this call.
  Overload 1 of 2, '(props: { defaultChecked?: boolean | undefined; defaultValue?: string | number | readonly string[] | undefined; suppressContentEditableWarning?: boolean | undefined; suppressHydrationWarning?: boolean | undefined; ... 268 more ...; value?: string | ... 2 more ... | undefined; } & { ...; } & { ...; }): ReactElement<...>', gave the following error.
    Type '{}' is not assignable to type 'ReactNode'.
  Overload 2 of 2, '(props: StyledComponentPropsWithAs<"button", DefaultTheme, {}, never, "button", "button">): ReactElement<StyledComponentPropsWithAs<"button", DefaultTheme, {}, never, "button", "button">, string | JSXElementConstructor<...>>', gave the following error.
    Type '{}' is not assignable to type 'ReactNode'.ts(2769)
```

I don't think it's breaking change. Because `ReactNode` contains `{} (DO_NOT_USE_OR_YOU_WILL_BE_FIRED_EXPERIMENTAL_REACT_NODES)`. We should be good because `ReactNode` basically covers every case and I tested it in the demo when making this change.

>/**
> * For internal usage only.
> * Different release channels declare additional types of ReactNode this particular release channel accepts.
> * App or library types should never augment this interface.
> */
>interface DO_NOT_USE_OR_YOU_WILL_BE_FIRED_EXPERIMENTAL_REACT_NODES {}
>type ReactNode =
>    | ReactElement
>    | string
>    | number
>    | ReactFragment
>    | ReactPortal
>    | boolean
>    | null
>    | undefined
>    | DO_NOT_USE_OR_YOU_WILL_BE_FIRED_EXPERIMENTAL_REACT_NODES[keyof DO_NOT_USE_OR_YOU_WILL_BE_FIRED_EXPERIMENTAL_REACT_NODES];

Reply:
Nice! Good that you have validated already.